pkg/fileutils/fileutils_unix.go

```diff
@@ -14,7 +14,7 @@ import (
 // reading it via /proc filesystem.
 func GetTotalUsedFds() int {
 	if fds, err := ioutil.ReadDir(fmt.Sprintf("/proc/%d/fd", os.Getpid())); err != nil {
-		logrus.Errorf("Error opening /proc/%d/fd: %s", os.Getpid(), err)
+		logrus.Errorf(err)
```

Comment:
```suggestion
		logrus.Errorf("%v", err)
```
I think will make lint happy